fastparquet/writer.py

```diff
@@ -607,6 +604,31 @@ def write(filename, data, partitions=[0], encoding="PLAIN",
                               open_with)
 
 
+def partition_on_columns(data, columns, root_path, partname, fmd, sep,
+                         compression, encoding, open_with, mkdirs):
+    """Split each data division on columns, and write in structured directories
```

Comment:
Slight request to flip the terms terms partition and division.  In dask/dataframe partitions are the segments of dataframes themselves while divisions are the index values between dataframes.